### DISCUSSION TITLE
Which github repo is feeding the ldproxy docker hub page?

### DISCUSSION THREAD

**Discussion by swiss-knight** [ldproxy/ldproxy#923] [General]
Hello,

I'm facing some issues while trying to use the latest ldproxy docker image available at https://hub.docker.com/r/iide/ldproxy/ and I was quickly wondering if the [current repository](https://github.com/interactive-instruments/ldproxy) is the one which actually builds this docker image.

But because there is no Docker related files in the source code, it seems it's not the case.

Hence my question: which repository, if any, is used to actually build and push the ldproxy docker image that is publicly available?

I wasn't able to find a link to a github page on the ldproxy docker hub webpage itself, as for example this is the case for PostgreSQL; its docker hub [page](https://hub.docker.com/_/postgres) has a straightforward link to https://hub.docker.com/_/postgres which in return has a link to https://github.com/docker-library/postgres. 

It would be great to have the same kind of explicit relation for ldproxy so that people can contribute / debug things themselves if they want to.

Thanks for reading.
Warm regards.

**Comment by azahnen** [comment]
Hi there,

yes, https://github.com/interactive-instruments/ldproxy is the source of the docker image. You can see the docker build in the ci scripts [here](https://github.com/interactive-instruments/ldproxy/blob/4e5e3afd9b3681556858cdca0056b35e20d07b02/.woodpecker/snapshot.yml#L54) and latest build of `iide/ldproxy:next` on our ci server [here](https://ci.ldproxy.net/interactive-instruments/ldproxy/build/1853/10).

The `Dockerfile` which I guess you are missing is part of our Gradle Plugins since it is used for multiple applications. It is actually pretty basic, it is just `eclipse-temurin:11-jre` with the application copied into. You can see it [here](https://github.com/interactive-instruments/xtraplatform-build/blob/0c26d217aeaadae8dac0b791c0077a6a2b6edd12/xtraplatform-modules/src/main/groovy/de/interactive_instruments/xtraplatform/ApplicationPlugin.groovy#L232).
